ooasp/encodings/ooasp_aux_config.lp

```diff
@@ -7,6 +7,8 @@
 
 ooasp_domain(cls,new_object).
 
+ooasp_unused(new_object):- #false: ooasp_isa(C, new_object).
```

Comment:
@susuhahnml please add a comment to explain this rule, it looks rather unusual :)

Reply:
I removed it. It was for a feature of the UI that we did not use.

Reply:
Thank you.
Could you still briefly explain how it works, please? I've seen similar rules in other places in this project, but I've never seen rules like it before. Is the body similar to `not ooasp_isa(_, new_object)`, or does it mean something different?